.github/workflows/build.yml

```diff
@@ -387,7 +387,8 @@ jobs:
           node --test test/config.test.mjs
         working-directory: example
       - name: Test
-        if: ${{ steps.affected.outputs.macos != '' && github.event_name != 'schedule' }}
+        # Temporarily disabled due to random failures
+        if: false # ${{ steps.affected.outputs.macos != '' && github.event_name != 'schedule' }}
```

Comment:
Do you normally track with a Github issue or a TODO or anything like that?

Reply:
https://github.com/microsoft/react-native-test-app/issues/2180